changelogs/fragments/proxy.yml

```diff
@@ -0,0 +1,18 @@
+breaking_changes:
```

Comment:
Same as for zabbix_javagateway changelog fragments, information that is not relevant to the end user is not supposed to be here. Everything else needs to be prepended with the name of this role otherwise CHANGELOG.md will hardly make sense

Reply:
Updates Made